cms/templates/js/tag-list.underscore

```diff
@@ -0,0 +1,33 @@
+<% if (tags !== null) { %>
+    <div class="wrapper wrapper-tags">
+        <div class="wrapper-tag-header">
+            <span>
+                <span class="tag-title label">
+                    <%- gettext("Tags") %>
+                </span>
+                <span class="count-badge">
+                    <%- tags.count %>
+                </span>
+            </span>
+            <span class="icon fa fa-caret-down" aria-hidden="true"></span>
+        </div>
+        <div class="wrapper wrapper-tag-content is-hidden">
+            <div class="content-taxonomies">
+                <% for (var i = 0; i < tags.taxonomies.length; i++) {
+                    var taxonomy = tags.taxonomies[i];
+                %>
+                    <div class="tagging-label tagging-label-tax-<%- taxonomy.id %>">
+                        <a id="tax-<%- taxonomy.id %>">
+                            <%- taxonomy.value %> (<%- taxonomy.count %>)
+                        </a>
+                        <span class="icon fa fa-caret-down" aria-hidden="true" id="tax-<%- taxonomy.id %>"></span>
```

Comment:
For me, the caret sometimes appears on the wrong line:

![Screenshot 2023-11-14 at 1 43 23 PM](https://github.com/openedx/edx-platform/assets/945577/26a45ba9-221b-4993-9104-ec7a6d51b421)

You can fix it with some styles like this:
![Screenshot 2023-11-14 at 1 44 00 PM](https://github.com/openedx/edx-platform/assets/945577/df1312cb-daad-46a8-8181-77ce6ab889d8)

![Screenshot 2023-11-14 at 1 44 19 PM](https://github.com/openedx/edx-platform/assets/945577/e0c5befe-c795-4d46-a349-804a7b6facfc)

Reply:
Though @ali-hugo can you please comment on how this is supposed to look/work?

Reply:
There is also an issue with the color of the tags. Some are blue and some are black. 

![Screenshot 2023-11-14 at 1 49 06 PM](https://github.com/openedx/edx-platform/assets/945577/c9bd5afc-1261-46d1-9fff-254b1cf48984)

In Firma, all the tag text is the same color (blue). Though personally now that I can try it out, I find it a bit confusing that the text is blue but not a link.

Reply:
Thanks for the ping @bradenmacdonald. It's exciting to see this coming together @ChrisChV!

I think the way @bradenmacdonald has truncated the text in the screenshot below is the best option:

![282939533-e0c5befe-c795-4d46-a349-804a7b6facfc](https://github.com/openedx/edx-platform/assets/88187719/62926bfd-34e0-47a4-9a02-1acefbd66c2d)

---

> In Figma, all the tag text is the same color (blue). Though personally now that I can try it out, I find it a bit confusing that the text is blue but not a link.

The tags are actually all supposed to be black (i.e. dark gray). The text is only supposed to be blue upon hover. @ChrisChV please make all the tags dark gray and only use blue for the hover state. 

---

Lastly, the styling of the tags does not quite match the styling in the design. There was quite a bit of back-and-forth on this with the 2U design team, so it would be great if you could update the tags to match the screenshot below (the color of the outlines should all be #F2F0EF, and the icons for the implicit and explicit tags should be different):

<img width="673" alt="Screenshot 2023-11-15 at 14 26 41" src="https://github.com/openedx/edx-platform/assets/88187719/6ce13165-ace7-497c-9cc2-f33ac8dfdb40">

Let me know if I should add this feedback about the tag styling to a different ticket.

Reply:
Thanks @bradenmacdonald @ali-hugo!

> I think the way @bradenmacdonald has truncated the text in the screenshot below is the best option
> The tags are actually all supposed to be black (i.e. dark gray). The text is only supposed to be blue upon hover.

I updated both:

![Captura desde 2023-11-15 10-41-55](https://github.com/openedx/edx-platform/assets/11827305/0325ed42-dd75-477d-acd4-f313a280c3cc)
![Captura desde 2023-11-15 10-42-17](https://github.com/openedx/edx-platform/assets/11827305/edeac961-0bb5-4397-991d-0cb8ec61c340)

> Lastly, the styling of the tags does not quite match the styling in the design. There was quite a bit of back-and-forth on this with the 2U design team, so it would be great if you could update the tags to match the screenshot below (the color of the outlines should all be #F2F0EF, and the icons for the implicit and explicit tags should be different):

About this, the drawer is part of https://github.com/openedx/frontend-app-course-authoring/pull/654  CC @yusuf-musleh